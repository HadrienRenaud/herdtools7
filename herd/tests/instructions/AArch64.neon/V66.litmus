AArch64 V66
(* Tests LDAPUR/STLUR single structure with 32 bit lanes *)
{
int x=1; int y;
0:X0=x; 0:X1=y;
}
 P0             ;
 LDAPUR S0,[X0] ;
 STLUR S0,[X1]  ;
forall(y=1)
