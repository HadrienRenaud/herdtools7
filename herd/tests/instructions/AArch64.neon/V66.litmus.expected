Test V66 Required
States 1
[y]=1;
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall ([y]=1)
Observation V66 Always 1 0
Hash=5d98d6e849d5a50e323705f8e6866979

